pkgs/by-name/di/direwolf/package.nix

```diff
@@ -5,25 +5,27 @@
   cmake,
   alsa-lib,
   gpsd,
-  gpsdSupport ? false,
-  hamlib,
+  gpsdSupport ? true,
```

Comment:
This was changed from its previous value. Is there a reason this was false by default?

Reply:
see: https://github.com/NixOS/nixpkgs/pull/406106#issuecomment-2874299171